docs/user-guide/federation/index.md

```diff
@@ -46,3 +46,13 @@ The following guides explain some of the resources in detail:
 
 [API reference docs](/federation/docs/api-reference/readme/) lists all the
 resources supported by federation apiserver.
+
+## Cascading deletion
+
+kubernetes v1.5 includes alpha support of DeleteOptions.OrphanDependents for
```

Comment:
I would phrase this as "Kubernetes v1.5 adds alpha support for cascading deletion of federated resources. This can be achieved by setting `DeleteOptions.Orphandependents` to `false`. ...". What I mean is, it will be somewhat more user-readable if the sentence says "cascading deletion" and it then says how to do it than directly mentioning the API mechanics.

Also, should we use the JSON name instead of the Go struct name that is capitalized, i.e. `orphanDependents` instead of `OrphanDependents`?